teuthology/openstack/__init__.py

```diff
@@ -454,7 +465,7 @@ def __flavor_wrapper(self, min, good, hint, arch):
         select = None
         if self.get_provider() == 'ovh':
             log.debug("Looking for a match among the El Cheapo flavors...")
-            select = '^vps-ssd-'
+            select = '^(vps-ssd|s1)-'
```

Comment:
@kshtsk Did you want to reverse the order here?

Reply:
@smithfarm yes... probably this can be implemented with ECP feature.

Reply:
@kshtsk Good idea. Then this PR could be closed.